API_doc/Communication.md

```diff
@@ -4,14 +4,20 @@
 
 All requests must contain **REQUEST_NAME**, containing the type of action they want to do.
 Each response contain at least **REQUEST_STATE** (see below).
+All the local settings are applied after the config inclusions.
 
 ### Request types
 | Request Name | Request Description | Additional Argument(s) | Additional Returned Value(s) |
 | -------- | -------- | -------- | -------- |
 |*CONFIG_CREATE*|Create a new config.|**CONFIG_NAME**|**CONFIG_KEY**<br>**READONLY_CONFIG_KEY**|
 |*CONFIG_LOAD*| Load an existing config. |**CONFIG_KEY** *or* **READONLY_CONFIG_KEY**| **CONFIG_NAME**<br>**CONFIG_ID**|
 |*CONFIG_UNLOAD*| Unload config. |**CONFIG_ID**|*none*|
-|*CONFIG_INCLUDE*| Include a config in another |**CONFIG_ID**<br>**SRC**<br>**DEST**|*none*|
+|*CONFIG_DELETE*| Delete config. |**CONFIG_ID**|*none*|
+|*CONFIG_GET_DEPS*| Get the ordered list of all included configs. |**CONFIG_ID**|**DEPS** (array of READONLY_CONFIG_KEY)|
+|*CONFIG_GET_SETTINGS*| Get the list of all local settings. |**CONFIG_ID**|**SETTINGS** (map of settings : "SETTING_NAME" -> "SETTING_VALUE")|
```

Comment:
Maybe we should return only the setting names? I don't think this is a major deal-breaker but if anyone else thinks so too we should change it.

Reply:
I think that for inspection it's better to have only one request for all. Else, you'll have to maque one request by setting, it's heavy

Reply:
I agree with @Gaspard-- that sometimes  you could could care only if some settings are present whatever their value, so to have a map of setting as a return value would be kinda heavy for this type of situation, what I would do is to add an additionnal argument to specify if you want the values as well

Reply:
So, @Gaspard-- if we had a boolean to CONFIG_GET_SETTINGS specifying if we want to get the values or not, is it okay ?

Reply:
I'd rather have 2 queries in that case. Seems simpler and cleaner.